server/data/data-jmap-postgres/src/main/java/org/apache/james/jmap/postgres/change/PostgresEmailChangeModule.java

```diff
@@ -0,0 +1,64 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.jmap.postgres.change;
+
+import static org.apache.james.jmap.postgres.change.PostgresEmailChangeModule.PostgresEmailChangeTable.TABLE;
+
+import java.time.OffsetDateTime;
+import java.util.UUID;
+
+import org.apache.james.backends.postgres.PostgresModule;
+import org.apache.james.backends.postgres.PostgresTable;
+import org.jooq.Field;
+import org.jooq.Record;
+import org.jooq.Table;
+import org.jooq.impl.DSL;
+import org.jooq.impl.SQLDataType;
+
+public interface PostgresEmailChangeModule {
+    interface PostgresEmailChangeTable {
+        Table<Record> TABLE_NAME = DSL.table("email_change");
+
+        Field<String> ACCOUNT_ID = DSL.field("account_id", SQLDataType.VARCHAR.notNull());
+        Field<UUID> STATE = DSL.field("state", SQLDataType.UUID.notNull());
+        Field<OffsetDateTime> DATE = DSL.field("date", SQLDataType.TIMESTAMPWITHTIMEZONE.notNull());
+        Field<Boolean> IS_DELEGATED = DSL.field("is_delegated", SQLDataType.BOOLEAN.notNull());
```

Comment:
We can change the name to `is_shared`

tldr: In the grooming ticket, I copied DDL table from cassandra and forgot it.
In pass, we had a ticket for renaming it 
https://github.com/apache/james-project/pull/1605
https://github.com/linagora/tmail-backend/pull/729#discussion_r1228922984